dll/win32/ucrtbase/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+
+spec2def(ucrtbase.dll ucrtbase.spec ADD_IMPORTLIB)
+
+add_library(ucrtbase SHARED
+    stubs.c
+    ucrtbase_stubs.c
+    ucrtbase.def
+    ${REACTOS_SOURCE_DIR}/sdk/lib/crt/startup/tlsmcrt.c # HACK
```

Comment:
Why is it a hack?

Reply:
It shouldn't be here, and in fact it's not needed, at least for now. Removed it.